examples/grpc/common/pom.xml

```diff
@@ -0,0 +1,146 @@
+        </dependency>
+
+        <dependency>
+          <groupId>io.grpc</groupId>
+          <artifactId>grpc-netty</artifactId>
+        </dependency>
+        <dependency>
+          <groupId>io.grpc</groupId>
+          <artifactId>grpc-stub</artifactId>
+        </dependency>
+        <dependency>
+          <groupId>io.grpc</groupId>
+          <artifactId>grpc-services</artifactId>
+        </dependency>
+        <dependency>
+          <groupId>io.grpc</groupId>
+          <artifactId>grpc-protobuf</artifactId>
+        </dependency>
+
+        <dependency>
+          <groupId>io.helidon.common</groupId>
+          <artifactId>helidon-common</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>io.helidon.config</groupId>
+          <artifactId>helidon-config-yaml</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>io.helidon.health</groupId>
+          <artifactId>helidon-health</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>io.helidon.metrics</groupId>
+          <artifactId>helidon-metrics</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>io.helidon.tracing</groupId>
+          <artifactId>helidon-tracing</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>io.helidon.tracing</groupId>
+          <artifactId>helidon-tracing-zipkin</artifactId>
+          <version>${project.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>io.opentracing.contrib</groupId>
+          <artifactId>opentracing-grpc</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+      <extensions>
+          <extension>
+            <groupId>kr.motd.maven</groupId>
+            <artifactId>os-maven-plugin</artifactId>
+            <version>1.5.0.Final</version>
+          </extension>
+        </extensions>
+        <plugins>
+          <plugin>
+              <groupId>org.apache.maven.plugins</groupId>
+              <artifactId>maven-compiler-plugin</artifactId>
+              <version>3.7.0</version>
+              <configuration>
+                  <source>1.8</source>
+                  <target>1.8</target>
+                  <compilerArgs>
+                      <arg>-Xlint:unchecked</arg>
+                  </compilerArgs>
+              </configuration>
+          </plugin>
+
+          <plugin>
+            <groupId>org.xolstice.maven.plugins</groupId>
+            <artifactId>protobuf-maven-plugin</artifactId>
+            <version>0.5.1</version>
+            <configuration>
+              <protocArtifact>com.google.protobuf:protoc:3.5.1-1:exe:${os.detected.classifier}</protocArtifact>
```

Comment:
If this configuration is common to all grpc example, then it should go under pluginManagement in exmaples/grpc/pom.xml

Reply:
It is also used by `config/etcd/pom.xml` so I'll move it to the top level pom where the plugin is declared.